src/Pages/Home/Home.js

```diff
@@ -0,0 +1,120 @@
+import "./Home.css";
+import { useState } from "react";
+import { useUserTodo } from "../../Context/useUserTodo";
+import { no_of_pending_task } from "../../Utils/no_of_pending_task";
+import { Modal } from "../../Components";
+import { useGetLocalData } from "../../Hooks/useGetLocalData";
+
+export const Home = () => {
+  useGetLocalData();
+  const { user_data, setUser_Data } = useUserTodo();
```

Comment:
either follow `lower_snake_case` or `camelCase` don't mix up both. ~~setUser_Data~~ `set_user_data` or `setUserData`. `camelCase` is convention

Reply:
Changed it..